meson.build

```diff
@@ -49,6 +49,7 @@ libnvme_dep = dependency('libnvme', version: '>=1.3', required: true,
                          fallback : ['libnvme', 'libnvme_dep'])
 libnvme_mi_dep = dependency('libnvme-mi', required: true,
                          fallback : ['libnvme', 'libnvme_mi_dep'])
+libxnvme_dep = dependency('xnvme', version: '>=0.2.0', required: true)
```

Comment:
This should be guarded by meson option setup option to disable it completely, see `json-c` as example. Some folks really don't like if we add hard dependencies.